content/news/2022-11-14-BGE-kickoff-meeting/index.md

```diff
@@ -0,0 +1,26 @@
+---
+title: BGE - Kickoff meeting
+date: "2022-11-14"
+tease: "The BGE project kick-off meeting was celebrated between November 14th and November 18th in Leiden."
+hide_tease: false
+tags: [BGE, biodiversity]
+subsites: [all-eu, esg]
+main_subsite: eu
+---
+
+From November 14 to November 18, the Naturalis Biodiversity Center in Leiden played host to an extraordinary event—the kick-off meeting of the Biodiversity Genome Europe (BGE) project. This momentous gathering brought together leading scientists, researchers, and conservationists from across Europe, marking the beginning of an ambitious initiative focused on unraveling the mysteries of biodiversity through genomics.
+
+![BGE kick-off meeting](BGE_image.jpg)
+
+The BGE project represents a collaborative endeavor involving numerous esteemed institutions and organizations dedicated to advancing our understanding of Earth's remarkable biodiversity. With genomics as its cornerstone, this groundbreaking project aims to extensively catalog and sequence the genomes of countless species, providing invaluable insights into their genetic makeup, evolutionary history, and ecological significance.
+
+The Naturalis Biodiversity Center, a renowned institution at the forefront of biodiversity research, provided a fitting backdrop for this momentous occasion. Its state-of-the-art facilities and expertise in genomic analysis made it an ideal host for the kick-off meeting. Participants had the opportunity to tour the center's laboratories and witness to witness firsthand the advanced computational infrastructure at Naturalis, which hosts a dedicated Galaxy server.
+
+During the meeting, working groups were formed, bringing together experts from different disciplines to collaborate on specific aspects of the project. These groups focused on topics such as data management, sample collection, sequencing techniques, and bioinformatics. Through lively discussions and knowledge sharing, participants charted a course for the project's future endeavors.
+
+![Kick-off meeting attendants](kickoff_people.jpeg)
+
+The collaboration between the BGE project and the Galaxy project extends beyond the kick-off meeting, as ongoing efforts will involve continuous development and refinement of bioinformatics workflows, tools, and resources. The Galaxy project will play a crucial role in ensuring the accessibility, scalability, and reproducibility of bioinformatics analyses, empowering researchers across Europe to delve into the intricate genetic tapestry of biodiversity.
```

Comment:
link to the galaxyproject.org/projects/vgp site? Or maybe add a big button with "more information" that points to this site?